readme.txt

```diff
@@ -0,0 +1,67 @@
+=== Komoju Payments ===
+Contributors: degica
+Tags: WooCommerce,Payment Gateway,Komoju
+Requires at least: 5.3
+Tested up to: 5.4
+Stable tag: trunk
+Requires PHP: 7.2
+License: MIT
+License URI: https://directory.fsf.org/wiki/License:X11
+
+A WooCommerce plugin to receive payments with Komoju, Japan's leading provider for online and offline payments.
```

Comment:
Not sure if this is true, but it sounds good 😛 . If there's different copy that's preferrable to use let me know, but this field has a 150 character limit on it.

Reply:
Maybe something like the main page copy would be good - "Built for developers, KOMOJU is the easiest way to integrate Japanese payments". I think we can leave out the WooCommerce plugin part since thats implicit in it being on the plugin store

Reply:
The reason I left WooCommerce in there is because I was intending to upload it to wordpress.org (since WooCommerce is also a WP plugin) but that prompted me to check and WooCommerce _does_ have their own online store for plugins. I think it makes sense to start with the WooCommerce store and then bring it across to WordPress later if we feel we need.

I'm going to put this PR on hold for a minute while I figure out what the requirements for the WooCommerce store is.

Reply:
Oh yeah I was working under the assumption that we were going on that store 😅  sorry for the confusion.

Reply:
No this good 😄 , I'd rather figure this out now rather than after sinking in a bunch more time.

Reply:
As we found out yesterday WooCommerce isn't interested in more plugins for payment options, so for now we should target the wordpress.org store 😄 .